Makefile

```diff
@@ -50,9 +50,12 @@ clean-test: ## remove test and coverage artifacts
 lint: ## check style with flake8
 	flake8 graphql_ws tests
 
+format:
+	black ./graphql_ws ./tests
```

Comment:
I added a script to format the code locally. 
It helps solve flake8 reported issues. Let me know if this is out of the scope of this PR

Reply:
Makes sense to me 👍